Port Selenium JS `get text` tests to webdriver

shs96c opened the pull request:
The `Element Get Text` command says that implementations
must exactly match Selenium's existing text handling. To
ensure this, the tests from Selenium's atoms and webdriver
implementations are being ported to the WPT suite.

This PR is equivalent to the tests in //javascript/atoms/test

<!-- Reviewable:start -->

<!-- Reviewable:end -->

shs96c committed:
Port Selenium JS `get text` tests to webdriver

The `Element Get Text` command says that implementations
must exactly match Selenium's existing text handling. To
ensure this, the tests from Selenium's atoms and webdriver
implementations are being ported to the WPT suite.

This PR is equivalent to the tests in //javascript/atoms/test

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260580991)

# Firefox (nightly) #

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260580992)

# Sauce (safari) #

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260580999)

# Chrome (unstable) #

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260581000)

# Sauce (MicrosoftEdge) #

shs96c committed:
[webdriver]: Add attribution for selenium tests

After speaking with the SFC legal folks, who own the copyright
for the Selenium project, the following change was suggested:

> Under Apache 2.0, you can redistribute modified versions of
> Selenium files (including files ported into a different programming
> language) under a different license, including the W3C's 3-clause BSD
> license.  Be sure to add a note indicating that the files have been
> modified, and include the appropriate copyright/attribution notices (and
> patent/trademark, if applicable) for the original files.

I've added links to the original sources and to the original
Apache 2.0 license, so I think that's now covered properly.

w3c-bots commented:
# Build PENDING

Started: None
Finished: None


<h2>Failing Jobs</h2>
<ul>

<li>firefox:nightly</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17667)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/267262422)
